Dockerfile

```diff
@@ -1,19 +1,20 @@
 FROM {{%DRYDOCK_ORG%}}/microbase:{{%TAG%}}
 
-ENV REQPROC_PATH /home/shippable/reqProc
-ADD . $REQPROC_PATH
-RUN cd $REQPROC_PATH && npm install
+# Prefix all image ENVs with IMAGE_ so there are no confusions.
+ENV IMAGE_REQPROC_DIR /home/shippable/reqProc
+ADD . $IMAGE_REQPROC_DIR
+RUN cd $IMAGE_REQPROC_DIR && npm install
 
-ENV EXEC_TEMPLATES_PATH /home/shippable/execTemplates
-RUN mkdir -p $EXEC_TEMPLATES_PATH && \
+ENV IMAGE_EXEC_TEMPLATES_DIR /home/shippable/execTemplates
+RUN mkdir -p $IMAGE_EXEC_TEMPLATES_DIR && \
     wget https://github.com/Shippable/execTemplates/archive/{{%TAG%}}.tar.gz -O /tmp/execTemplates.tar.gz && \
-    tar -xzvf /tmp/execTemplates.tar.gz -C $EXEC_TEMPLATES_PATH --strip-components=1 && \
+    tar -xzvf /tmp/execTemplates.tar.gz -C $IMAGE_EXEC_TEMPLATES_DIR --strip-components=1 && \
     rm /tmp/execTemplates.tar.gz
 
-ENV REQEXEC_PATH /home/shippable/reqExec
-RUN mkdir -p $REQEXEC_PATH && \
-    wget https://s3.amazonaws.com/shippable-artifacts/reqExec/{{%TAG%}}/reqExec-{{%TAG%}}-{{%ARCHITECTURE%}}-{{%OS%}}.tar.gz -O /tmp/reqExec.tar.gz && \
-    tar -xzvf /tmp/reqExec.tar.gz -C $REQEXEC_PATH && \
+ENV IMAGE_REQEXEC_DIR /home/shippable/reqExec
```

Comment:
We are pulling `IMAGE_REQEXEC_DIR/x86_64/Ubuntu_16.04`. The rest of them should also be explicitly pulled once we have them in S3.